### DISCUSSION TITLE
Fiche à renseigner par le lycéen (création de son compte)

### DISCUSSION THREAD

**Discussion by SandrineRobert** [unistra/esup-immersup-demandes#8] [3- Création de compte pour venir en immersion]
**(1) Options à cocher sur la visibilité des immersions et du courriel**
Pourrait-on reformuler les phrases comme suit :
 - Rendre visibles **mes** inscriptions aux immersions aux lycéens **de mon** lycée (s'ils sont inscrits aux mêmes immersions) 
 - Rendre visible **mon** courriel aux lycéens **de mon**  lycée (s'ils sont inscrits aux mêmes immersions) 

**(2) Affichage du "statut" de la création de compte :** 
- peut-on le supprimer sur la 1ère page ? ou alors ajouter "validation en attente" comme sur la 2ème page ?
Cf fichiers joints :
![Fiche 1ère page](https://github.com/unistra/esup-immersup-demandes/assets/152433302/e25cc0d0-2f97-4b1d-83af-f1e9ca990518)
![Fiche 2è page](https://github.com/unistra/esup-immersup-demandes/assets/152433302/f0a50697-1d6e-42ac-b1b7-df6d434cbaad)

**Comment by efinkbeiner5** [comment]
Chiffrage : faible

**Comment by efinkbeiner5** [comment]
Bugs corrigé en version 3 et changement des libellés fait en v4
